From 746535f5d5ced7f1df4f1cb483621939ac8076e8 Mon Sep 17 00:00:00 2001
From: Juan Quintela <quintela@redhat.com>
Date: Tue, 14 Jan 2014 15:07:17 +0100
Subject: [PATCH 06/40] exec: drop useless #if

RH-Author: Juan Quintela <quintela@redhat.com>
Message-id: <1389712071-23303-7-git-send-email-quintela@redhat.com>
Patchwork-id: 56658
O-Subject: [RHEL7 qemu-kvm PATCH 06/40] exec: drop useless #if
Bugzilla: 997559
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Dr. David Alan Gilbert (git) <dgilbert@redhat.com>

From: Paolo Bonzini <pbonzini@redhat.com>

This code is only compiled for softmmu targets.

Reviewed-by: Richard Henderson <rth@twiddle.net>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit ae4e43e80f3e1e65d096ebbc643eed9d763e1441)
Signed-off-by: Juan Quintela <quintela@trasno.org>
---
 exec.c | 2 --
 1 file changed, 2 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 exec.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/exec.c b/exec.c
index 389888d..436dbe9 100644
--- a/exec.c
+++ b/exec.c
@@ -1483,10 +1483,8 @@ static void notdirty_mem_write(void *opaque, hwaddr ram_addr,
     int dirty_flags;
     dirty_flags = cpu_physical_memory_get_dirty_flags(ram_addr);
     if (!(dirty_flags & CODE_DIRTY_FLAG)) {
-#if !defined(CONFIG_USER_ONLY)
         tb_invalidate_phys_page_fast(ram_addr, size);
         dirty_flags = cpu_physical_memory_get_dirty_flags(ram_addr);
-#endif
     }
     switch (size) {
     case 1:
-- 
1.7.1

